price/pricing/price_updater.go

```diff
@@ -257,6 +257,12 @@ type LatestPrices struct {
 	PerCountry         map[string]*PriceHistory `json:"per_country"`
 	CurrentValidUntil  time.Time                `json:"current_valid_until"`
 	PreviousValidUntil time.Time                `json:"previous_valid_until"`
+	CurrentServerTime  time.Time                `json:"current_server_time"`
```

Comment:
as this is kinda sort "optional", we could add `omitempty` or something here, just in case we dont add it. Probably things would break if the code that relies on this suddenly got `1900-00-00 00:00:00` or something because we forgot to add `WithCurrentTime()`

Reply:
This will definitely be not optional from when we make changes in node :)